src/teleop/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -44,6 +44,7 @@
       <div class="box datagrid">
         <div class="auton-check">
           <AutonModeCheckbox ref="autonCheckbox" v-bind:name="autonButtonText" v-bind:color="autonButtonColor"  v-on:toggle="toggleAutonMode($event)"/>
+          <Checkbox ref="teleopCheckbox" v-bind:name="'Teleop Controls'" v-bind:color="teleopButtonColor" v-on:toggle="toggleTeleopMode($event)"/>
```

Comment:
Yeah the teleopButtonColor v-bind here needs to be removed. Also remove line 279, which also references teleopButtonColor.